src/common/types/annotations.js

```diff
@@ -61,7 +62,7 @@ export type Annotation = {
     status?: FeedItemStatus,
     target: Target,
     total_reply_count?: number,
-    type: 'annotation',
+    type: typeof FEED_ITEM_TYPE_ANNOTATION,
```

Comment:
Can we reverse the flow of types here? This file should be the source of truth for annotations-related types rather than the other way around.